README.md

```diff
@@ -91,11 +91,13 @@ The integration tests don't run by default. To run all the tests, including inte
 
 The tests needs a MySQL server with replication enabled.
 
-Ecco includes a Docker Compose definition that can be used for this.
+Ecco includes multiple Docker Compose definitions that can be used for this, one per supported MySQL version:
 
 Just start it before running the tests
 
-    docker-compose up
+* docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5-6.yml up
+* docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5-7.yml up
+* docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5-.8yml up
```

Comment:
```suggestion
* docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5.8.yml up
```
:D

Reply:
I took a look at the rendered version, perhaps it looks better with a block instead, like:

```
# MySQL 5.6
docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5-6.yml up

# MySQL 5.7
docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-5-7.yml up

# MySQL 8.0
docker-compose -f docker-compose-mysql-base.yml -f docker-compose-mysql-8-0.yml up
```

Reply:
Fixed now! :) 89c1296de64e949a34bb4205926284d9b510b636